src/SFA.DAS.ApprenticePortal.Web/Services/ApprenticeshipService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.ApprenticePortal.Web.Models;
+using SFA.DAS.ApprenticePortal.Web.Services.OuterApi;
+
+namespace SFA.DAS.ApprenticePortal.Web.Services
+{
+    public class ApprenticeshipService

```

Comment:
I wonder if we want an API that returns the Apprentice *and* the Apprenticeship, to save a HTTP call on every page load?

Reply:
I didn't want to fall into the trap of creating God objects, so it seems like a good idea to keep functionality in smaller classes with a defined propose.   Happy to discuss

Reply:
I agree in terms of the classes in the project.  I was thinking (out load) about reducing the number of HTTP calls required.  Probably not important for this PR, let's have a chat about it in general.